package-lock.json

```diff
@@ -0,0 +1,5 @@
+{
+  "name": "ieee-ghost",
+  "version": "1.0.0",
+  "lockfileVersion": 1
+}
```

Comment:
Looks like you accidentally did `npm install` in the main directory. This file can be deleted.